src/08-accessing.js

```diff
@@ -148,7 +164,9 @@ function getCityAndState(person) {}
  *  getFullAddress(person);
  *  //> "697 Pine Drive 2A, Staten Island, NY, 10306"
  */
-function getFullAddress(person) {}
+function getFullAddress(person) {
+  return `${person.address.street} ${person.address.unit}, ${person.address.city}, ${person.address.state}, ${person.address.zip}`
```

Comment:
For long stings like this consider placing the variable on their own lines to make it easier to read.  We can use multi line strings if we surround them with backticks.

Reply:
Hey Tim, I was under the impression that indenting the lines might change the format of how the code would return. Thank you for clarifying that it wouldn't. I certainly need to improve my formatting skills a bit.